R/add_CNSR.R

```diff
@@ -103,7 +103,8 @@ add_CNSR.ggsurvfit <- function(gg, shape = 3, size = 2, ...){
                                      y = est,
                                      color = strata),
                         shape = shape,
-                        size = size)
+                        size = size,
+                        show.legend = FALSE)
```

Comment:
We decided not to include this on purpose. The coloring should be sufficient without showing the censoring symbols. If you have a specific business need, we can reopen the discussion.

Reply:
I am not sure I understand this comment. After my other updates, the CNSR symbol legend was being added rather than integrated with the line color legend. I agree the color legend should be sufficient, which is now what shows.